__tests__/cmds/login.test.js

```diff
@@ -11,87 +16,69 @@ describe('rdme login', () => {
   afterEach(() => configStore.clear());
 
   it('should error if no project provided', () => {
-    return expect(cmd.run({})).rejects.toThrow('No project subdomain provided. Please use `--project`.');
+    return expect(cmd.run({})).rejects.toStrictEqual(
+      new Error('No project subdomain provided. Please use `--project`.')
+    );
   });
 
   it('should error if email is invalid', () => {
-    expect.assertions(1);
-    return expect(cmd.run({ project: 'subdomain', email: 'this-is-not-an-email' })).rejects.toThrow(
-      'You must provide a valid email address.'
+    return expect(cmd.run({ project: 'subdomain', email: 'this-is-not-an-email' })).rejects.toStrictEqual(
+      new Error('You must provide a valid email address.')
     );
   });
 
-  it('should post to /login on the API', () => {
+  it('should post to /login on the API', async () => {
     expect.assertions(3);
-    const email = 'dom@readme.io';
-    const password = '123456';
-    const project = 'subdomain';
     const apiKey = 'abcdefg';
 
-    const mock = nock(config.host).post('/api/v1/login', { email, password, project }).reply(200, { apiKey });
+    const mock = nock(config.get('host')).post('/api/v1/login', { email, password, project }).reply(200, { apiKey });
 
-    return cmd.run({ email, password, project }).then(() => {
-      mock.done();
-      expect(configStore.get('apiKey')).toBe(apiKey);
-      expect(configStore.get('email')).toBe(email);
-      expect(configStore.get('project')).toBe(project);
-      configStore.clear();
-    });
-  });
+    await cmd.run({ email, password, project });
```

Comment:
Should we check the resolved value here for the command output? https://github.com/readmeio/rdme/blob/54151a9f4ee17d4a74bac2fafd9d64d8c49a78ff/src/cmds/login.js#L74